lib/ExpandableListItem/ExpandableListItem.less

```diff
@@ -1,11 +1,12 @@
 /* ExpandableListItem Header*/
 .moon-expandable-list-item-header {
+  padding-right: @moon-spotlight-outset + 30px;
```

Comment:
@graynorton Is correct; it certainly needs to be a more specific rule for this type of override. On a more general note, I've been looking at this control's CSS quite a bit lately (for line height stuff) and I'm thoroughly convinced ExpandablePicker's CSS needs to be completely reworked. Its design and construction doesn't take advantage of LESS structuring and easily gets clobbered in cases like this, and has disappointingly long class names.